### DISCUSSION TITLE
Confusing workflow-controller-configmap comments

### DISCUSSION THREAD

**Discussion by zsolt-keseru-epam** [argoproj/argo-workflows#8603] [Q&A] [answered]
Hi!
Could someone please elaborate on the following comments?
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L195
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L149
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L162
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L178
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L182

The comments are mentioning v3.4 version many times, it's confusing because there's no v3.4 released yet.

**Comment by terrytangyuan** [comment, marked as answer]
It means the upcoming release v3.4 which is not available yet. You’d need to wait for it or use the latest dev version.
